app/models/food.rb

```diff
@@ -0,0 +1,19 @@
+require 'dotenv'
+require 'yelp'
+
+class Food
+  attr_reader :name, :url, :image_url, :phone, :address
+
+  def initialize(data)
+    raise
```

Comment:
Best to remove these before making a PR. If someone is trying to use your site, like a code reviewer, they will just get an error.